libraries/engage/page/widgets/ProductList/hooks.js

```diff
@@ -40,6 +42,8 @@ export const useProductListWidget = () => {
     showName = false,
     showPrice = false,
     showRating = false,
+    showHeadline = false,
+    headline = '',
```

Comment:
please remove the default value since it's not necessary. additionally it's kinda wrong, since `headline` is usually an object. so the default shouldn't be a string.